lib/modules/datasource/clojure/__snapshots__/index.spec.ts.snap

```diff
@@ -13,15 +13,24 @@ exports[`modules/datasource/clojure/index falls back to next registry url 1`] =
       "version": "0.0.1",
     },
     {
-      "releaseTimestamp": "2020-01-01T01:00:00.000Z",
```

Comment:
Do we need to add this function to the clojure datasource too, maybe also sbt-package also?